README.md

```diff
@@ -54,6 +54,17 @@ To test authorization for Management Service locally, make the following changes
 - Set AuthorizationConfig.Enabled=true in the config file that's being used
 - Issue requests to the app with the header User-Email: test-user@gojek.com
 
+#### c. Using XP
+
+To use XP, `replace` directives need to be used when consuming both the Management and Treatment API modules since the API modules uses local relative path.
```

Comment:
To use XP is a bit vague.. Can it be rephrased as, "to use the XP Go modules as API dependencies ..."